packages/cockpit/src/balance/total/index.js

```diff
@@ -1,6 +1,8 @@
 import buildRequest from './buildRequest'
 
-const total = client => (recipientId, query) =>
-  buildRequest(client, { recipientId, ...query })
+const total = client => (recipientId, query) => buildRequest(client, {
```

Comment:
fico em duvida sobre qual das duas opções ficam mais legiveis aqui nesse caso, seria legal mais pessoas opinarem
```suggestion
const total = client => (recipientId, query) => (
  buildRequest(client, {
    recipientId,
    ...query,
  })
)
```